### PR TITLE
Branch 161949540

### PR DESCRIPTION
**Pull request by caisq**

### CODE REVIEW

**Review by frankchn — APPROVED** [review]

**Review by ebrevdo — APPROVED** [review]

**Review by agarwal-ashish — APPROVED** [review]
Looks good for contrib/factorization changes.

File: tensorflow/core/ops/ops.pbtxt [hunk 1]
```diff
@@ -28468,33 +28468,6 @@ op {
   is_stateful: true
 }
 op {
-  name: "LMDBReader"
```
Comment (alextp) [inline tensorflow/core/ops/ops.pbtxt]: Is it safe to remove an op?

**Comment by caisq** [comment]
@alextp these deletions to ops.pbtxt are auto-generated.

Looks like @jhseu should know about the underlying change.

**Comment by caisq** [comment]
@allenlavoie FYI, tensorflow/contrib/timeseries/python/timeseries:state_management_test is segfaulting in the PIP build:
https://ci.tensorflow.org/job/tensorflow-pull-requests-cpu-python3/5400/consoleFull

@tensorflow-jenkins test this please

testing again to see whether it is reproducible.

File: tensorflow/core/ops/ops.pbtxt (hunk 1 above)
Comment (jhseu) [inline tensorflow/core/ops/ops.pbtxt]: Hmm, yeah, this is a surprising deletion. Let's track down why it happened?

Comment (jhseu) [inline tensorflow/core/ops/ops.pbtxt]: Ah, it looks like it just removed a duplicate entry, so this seems fine.

**Review by jhseu — APPROVED** [review]

**Review by alextp — APPROVED** [review]

File: tensorflow/core/ops/ops.pbtxt (hunk 1 above)
Comment (caisq) [inline tensorflow/core/ops/ops.pbtxt]: Thanks for investigating this, @jhseu

**Comment by caisq** [comment]
@allenlavoie Looks the failure in state_management_test is random. There may be other tests in contrib/timeseries that have the same problem. We will keep an eye on them. I'm merging this PR now.

**Comment by allenlavoie** [comment]
Yeah, I can reproduce the same kind of flakiness for that test locally (passes, but segfaults after) without this PR. Odd since we're not using any timeseries-specific custom ops and it's not at all flaky internally. I'll try to track it down.

**Pull request merged**